geoapify/client.py

```diff
@@ -1,16 +1,23 @@
 import logging
-import math
-import time
-from typing import Dict, List, Tuple, Any
+import warnings
+from typing import Dict, List, Tuple
 
 import requests
 
+from geoapify.batch import BatchClient
+
+HEADERS = {'Accept': 'application/json', 'Content-Type': 'application/json'}
```

Comment:
Just out of interest:
Why do you define the headers variable like that and not as an attribute of the class?
The same applies for '_url_place_details, _url_geocode...' . When do you typically define a variable as an attribute and when not?

Reply:
Yes, this is weird.

I used to import this HEADERS variable also in client.py but not doing anymore. Will change that.

Re _url_*: 

Note that

class SomeClass:
   x = <some-val>

is equivalent to

class SomeClass:
   def __init__(self):
       self.x = <some-val>

I am using the former usually when I am setting a simple constant (float or string etc) while using the latter in all other cases. Don't ask why.